pkg/agent/datamodel/types.go

```diff
@@ -1054,9 +1055,11 @@ func (a *AgentPoolProfile) GetKubernetesLabels(rg string, deprecated bool) strin
 		buf.WriteString(",kubernetes.io/role=agent")
 	}
 	buf.WriteString(fmt.Sprintf(",agentpool=%s", a.Name))
+	storagetier, _ := common.GetStorageAccountType(a.VMSize)
 	if strings.EqualFold(a.StorageProfile, ManagedDisks) {
-		storagetier, _ := common.GetStorageAccountType(a.VMSize)
 		buf.WriteString(fmt.Sprintf(",storageprofile=managed,storagetier=%s", storagetier))
+	} else if strings.EqualFold(a.StorageProfile, Ephemeral) {
+		buf.WriteString(fmt.Sprintf(",storageprofile=ephemeral,storagetier=Standard_LRS"))
```

Comment:
why not storagetier=%s but always Standard_LRS?

Reply:
also is it possible to add a unit test, or modify an existing one? Current unit test is simple, it generates the cloud init files and cse and assert the content on them .

Reply:
> why not storagetier=%s but always Standard_LRS?

yup can do that


> also is it possible to add a unit test, or modify an existing one? Current unit test is simple, it generates the cloud init files and cse and assert the content on them .

let me dig around and see where i can add one.